### DISCUSSION TITLE
Aidex - When Aidex is used, number and arrow is not displayed

### DISCUSSION THREAD

**Discussion by andyrozman** [NightscoutFoundation/xDrip#2469] [Q&A]
If Aidex is used as source, Number and arrows are not displayed... 

Is this configuration problem (something not configured correctly) or is problem with my integration... This was my 1st help to Xdrip project in last 7 years (I did something small about 8 yrs ago I think) and I might have done something wrong... 

Any help with this would be welcome

**Comment by Navid200** [comment]
Is this something that is new since your PR was merged?
Or, this problem existed even when you submitted your PR?

**Comment by andyrozman** [comment]
It existed before. I could never get it to display that data (number and arrow), but since I was running on test device, I didn't know if this is configuration issue or something else (its been about 8 years since I first installed Xdrip, and after that I aways exported my data and imported, when I switched phones)...

**Reply by Navid200** [reply]
This is your PR, right?  https://github.com/NightscoutFoundation/xDrip/pull/2312
Are you saying the capability that PR added to xDrip is useless?
Did you not test before submitting the PR?

I cannot do anything about this particular part of xDrip.  But, I suspect the developers who may be able to help may have similar questions.

Do you mind looking at this from our perspective and help us understand exactly what you are describing?
For example, you talk about 8 years ago when you opened that PR only a few months ago.  Why would you talk about 8 years ago?

**Reply by jiresongit** [reply]
It is very useful even as is. It shows the numbers on the lock screen, in the status bar, in nightscout and on my smart watch, just not in the main xdrip app.

**Reply by jamorham** [reply]
please try today's nightly which should have a fix. See issue https://github.com/NightscoutFoundation/xDrip/issues/2470 for more information.

**Reply by jiresongit** [reply]
Yes! Sensor expired earlier today but can already see it's working. Thx 
![Screenshot_2022-11-16-21-19-57-222_com eveningoutpost dexdrip](https://user-images.githubusercontent.com/78233260/202296950-2626345d-0cd8-4788-b5cb-4bc8803ac322.jpg)

**Comment by andyrozman** [comment]
You can see the line and it has correct data, you just can't see the number in right corner and arrow. 

I was trying to explain that this is my first contribution to xDrip after quite a while, so I don't know the code. Adding this functionality was supposed to be pretty straightforward thing (and it was, or at least it looked so). After I added PR I had to make several changes as prompted by other developers, but nobody said that I am adding data wrong, so from my perspective everything was ok, and number not beeing displayed was configuration problem from my side.

Attached it picture how it looks:

![image](https://user-images.githubusercontent.com/5500133/202253166-bbb6cacd-c33e-46a6-964c-ca5a4f08b9d1.png)
